clang/test/SemaSYCL/sycl-device.cpp

```diff
@@ -1,4 +1,5 @@
 // RUN: %clang_cc1 -fsycl-is-device -fsyntax-only -verify %s
+// RUN: %clang_cc1 -fsycl-is-host -fsyntax-only -verify -DHOST %s
```

Comment:
NO_SYCL is not defined for this run, so this will only test the first part of the existing test.

Reply:
Nice catch. Thanks! Fixed it.